### PR TITLE
Create aws-cli container.yaml. OpenFOAM modules update

### PR DESCRIPTION
**Pull request by AlexisEspinosaGayosso**
Adding an amazon/aws-cli module

### CODE REVIEW

File: quay.io/pawsey/openfoam-org/aliases/10.yaml [hunk 1]
```diff
@@ -0,0 +1,186 @@
+aliases:
+  PDRFoam: PDRFoam
```
Comment (vsoch) [inline quay.io/pawsey/openfoam-org/aliases/10.yaml:2]: Aliases should always be full paths

Comment (AlexisEspinosaGayosso) [reply quay.io/pawsey/openfoam-org/aliases/10.yaml]: Thanks a lot @vsoch.
The *.yaml files for the aliases now refer to their full paths. Not only for `10.yaml` but for all the versions.

**Comment by vsoch** [comment]
@marcodelapierre could you do a quick review?

File: amazon/aws-cli/container.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+docker: amazon/aws-cli
+url: https://hub.docker.com/r/amazon/aws-cli
+maintainer: 'Amazon Web Services'
```
Comment (marcodelapierre) [inline amazon/aws-cli/container.yaml:3]: The maintainer value should be the github handle (i.e. username) of the yaml recipe creator

File: amazon/aws-cli/container.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+description: The AWS Command Line Interface (AWS CLI) is a unified tool to manage your AWS services.
+latest:
+    2.13.0: sha256:e119d71e8a39a6a3dec5f60f06b090dcdb7adb1b463ae6e9c1cb55d52a05d418
+tags:
+    2.13.0: sha256:e119d71e8a39a6a3dec5f60f06b090dcdb7adb1b463ae6e9c1cb55d52a05d418
+    latest: sha256:e119d71e8a39a6a3dec5f60f06b090dcdb7adb1b463ae6e9c1cb55d52a05d418
```
Comment (marcodelapierre) [inline amazon/aws-cli/container.yaml:9]: I would avoid adding the `latest` tag in the list of tags. 
The `latest` key in the yaml serves the purpose.
Right, @vsoch ?

**Review by marcodelapierre — CHANGES_REQUESTED** [review]
back from Europe and from leave!

here is my review, I am requesting just a couple of edits.

File: amazon/aws-cli/container.yaml (hunk 3 above)
Comment (vsoch) [reply amazon/aws-cli/container.yaml:9]: yes!

Comment (AlexisEspinosaGayosso) [reply amazon/aws-cli/container.yaml:9]: Thanks for this @marcodelapierre . I'll go over this again internally at Pawsey, let you review over there and then resubmit a new pull request into the original repo (here).

**Comment by marcodelapierre** [comment]
oh, and one more comment from @SarahBeecroft : 
the awscli recipe should also have:
```
features:
  home: true
```

**Pull request closed without merging**